lib/cache/FileCachePlugin.js

```diff
@@ -326,7 +326,7 @@ class FileCachePlugin {
 			if (store === "pack") {
 				promise = promise.then(serializePack);
 			}
-			return promise;
+			return /** @type {any} */ (promise);
```

Comment:
This is weird

Reply:
True but `Promise.all` returns `Promise<any[]>` which is not compatible with `Promise<void>` (and cannot be cast to it)